.gitignore

```diff
@@ -48,3 +48,5 @@ package-lock.json
 
 css/styles.css
 
+.vscode
```

Comment:
Please revert this and add it it your personal global gitignore; individual editors’ files shouldn’t force a change in every repo that individual uses.